miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/view/MiniAppViewImpl.kt

```diff
@@ -65,4 +66,15 @@ internal class MiniAppViewImpl(
             }
         }
     }
+
+    override fun sendJsonToMiniApp(message: String) {
+        message.let {
+            if (it.isNotBlank()) {
+                miniAppParameters.config.miniAppMessageBridge.dispatchNativeEvent(
+                    NativeEventType.MINIAPP_RECEIVE_JSON_INFO,
+                    message
+                )
+            }
```

Comment:
What if it is a blank ? We should return some error to Host app in that case as well right ?

Reply:
thanks, either the js SDK side will take this validation or.. if it is in here, I made the adjustment then.